astro.config.ts

```diff
@@ -4,13 +4,17 @@ import astroI18next from "astro-i18next";
 import sitemap from "@astrojs/sitemap";
 import compress from "astro-compress";
 import purgecss from "astro-purgecss";
+import rehypeAutolinkHeadings from 'rehype-autolink-headings';
+import { fromHtml } from 'hast-util-from-html'
+
 
 // https://astro.build/config
 export default defineConfig({
 	site: "https://quiltmc.org",
 	integrations: [mdx(), astroI18next(), sitemap(), compress(), purgecss()],
 	markdown: {
 		syntaxHighlight: "prism",
+		rehypePlugins: [[rehypeAutolinkHeadings, { content: fromHtml('<span class="header-anchor-container icon pl-3"><i class="header-anchor fas fa-lg fa-link has-text-grey-dark is-size-5"></i></span>'), behavior: "append" }]],
```

Comment:
`grey-dark` is *way* too dark for this — in fact, [it fails the WCAG color contrast standard](https://webaim.org/resources/contrastchecker/?fcolor=4A4A4A&bcolor=271F33) by quite a wide margin and we have to choose a brighter color for accessibility's sake

Reply:
I would probably use the `link` colour.